txnsync/emulatorTimer_test.go

```diff
@@ -92,6 +94,7 @@ func (g *guidedClock) Advance(adv time.Duration) {
 		ch       chan time.Time
 	}
 	expiredClocks := []entryStruct{}
+	g.mu.Lock()
```

Comment:
I wonder why I didn't ran into these when running it with "-race". but it's perfectly fine regardless.